src/sandbox/components/SandboxContent.svelte

```diff
@@ -109,11 +109,27 @@
     const container = document.querySelector(`#sheet-${escapeId(contentId)} .grid__container`);
     if (container) container.focus();
   }
-
 </script>
 
 <svelte:window
   on:keydown={kbListen}
+  on:resize={() => {
+    if ($contextMenuOpen) {
+      const windowHeight = document.documentElement.clientHeight;
+      const menu = document.querySelector("#ellx-context-menu");
+      const menuStyle = menu.style;
+
+      menuStyle.height = "auto"; // Drop style to calculate menu height
+
+      if (windowHeight < menu.clientHeight) {
+        menuStyle.height = "100vh"
+        menuStyle.overflowY = "scroll"
+      }
+      else {
+        menuStyle.overflowY = "hidden";
+      }
+    }
+  }}
```

Comment:
doesn't height 100% and overflowY "auto" not work? Why do we need this JS hack?

Reply:
Yeah, it's a bit hackish, but actually, I haven't found more simple solution that properly renders context menu and reacts on browser resize.

Reply:
height 100% and overflowY "auto"  ?